### PR TITLE
eks clusters migration

### PR DESCRIPTION
**Pull request by gurevichdmitry**
### Summary of your changes

This PR includes the following changes:

- Addition of configuration files related to the EKS clusters created in the AWS test account.
- Updates to the test infrastructure provisioning Helm charts.

Successful [run](https://github.com/gurevichdmitry/cloudbeat/actions/runs/5584554029) on my fork.

[Report](http://csp-allure-reports.s3.amazonaws.com/allure_reports/cloudbeat/main/5584554029/index.html)

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-test-configmap.yml [hunk 1]
```diff
@@ -10,6 +10,10 @@ data:
   ES_PASSWORD: "changeme"
   ES_HOST: "elasticsearch-master.kube-system"
   USE_DOCKER: "false"
+  EKS_CONFIG_1_NODE_1: {{ .Values.testData.conf1Node1}}
+  EKS_CONFIG_1_NODE_2: {{ .Values.testData.conf1Node2}}
+  EKS_CONFIG_2_NODE_1: {{ .Values.testData.conf2Node1}}
+  EKS_CONFIG_2_NODE_2: {{ .Values.testData.conf2Node2}}
```
Comment (olegsu) [inline tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-test-configmap.yml:16]: Use `{{ range }}` for iteration
https://helm.sh/docs/chart_template_guide/control_structures/#looping-with-the-range-action

File: tests/deploy/values/ci-eks-config-1.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ image:
 
 cloudbeat:
   deploy: false
+  repository: 704479110758.dkr.ecr.eu-west-2.amazonaws.com/cloudbeat
```
Comment (olegsu) [inline tests/deploy/values/ci-eks-config-1.yml:11]: is this the right account id?

File: tests/deploy/values/ci-eks-config-2.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ image:
 
 cloudbeat:
   deploy: false
+  repository: 704479110758.dkr.ecr.eu-west-2.amazonaws.com/cloudbeat
```
Comment (olegsu) [inline tests/deploy/values/ci-eks-config-2.yml:11]: is this the right account id?

File: tests/deploy/values/ci-eks-config-1.yml (hunk 2 above)
Comment (gurevichdmitry) [reply tests/deploy/values/ci-eks-config-1.yml:11]: Yes, this is currently needed for backward compatibility

File: tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-test-configmap.yml (hunk 1 above)
Comment (gurevichdmitry) [reply tests/deploy/k8s-cloudbeat-tests/templates/cloudbeat-test-configmap.yml]: `range` has been added to the `configmap`

File: .github/workflows/eks-ci.yml [hunk 4]
```diff
@@ -16,16 +16,16 @@ on:
...
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_TEST_ACC }}
   GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   CONTAINER_SUFFIX: ${{ github.run_id }}
   TEST_TARGETS_DEFAULT: '["pre_merge"]'
   AWS_REGION: eu-west-2
   REPORTS_DIR: tests/allure/results/
   KUBE_NAMESPACE: kube-system
   CI_ELASTIC_AGENT_DOCKER_TAG: "8.7.0-SNAPSHOT"
-  CI_ELASTIC_AGENT_DOCKER_IMAGE: "704479110758.dkr.ecr.eu-west-2.amazonaws.com/elastic-agent"
+  CI_ELASTIC_AGENT_DOCKER_IMAGE: "391946104644.dkr.ecr.eu-west-2.amazonaws.com/elastic-agent"
```
Comment (oren-zohar) [inline .github/workflows/eks-ci.yml:28]: should we bump the docker tag?

Comment (oren-zohar) [reply .github/workflows/eks-ci.yml:28]: also, why snapshot?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: .github/workflows/eks-ci.yml (hunk 4 above)
Comment (gurevichdmitry) [reply .github/workflows/eks-ci.yml:28]: Yes, we can update the docker tag to the latest version. These variables (`CI_ELASTIC_AGENT_DOCKER_TAG, CI_ELASTIC_AGENT_DOCKER_IMAGE )` are used to upload the latest agent docker snapshot with the latest Cloudbeat to the ECR. Initially, we had plans to implement integration tests with a standalone agent for EKS, just like we did for K8s. However, due to prioritization, this implementation was not given enough priority, and the uploaded image is currently not being used.

**Review by oren-zohar — APPROVED** [review]

**Pull request merged**